### PR TITLE
Building script and instructions for Docker image with OpenTelemetry …

### PR DESCRIPTION
**Pull request by piotr-sumo**
…jars

### CODE REVIEW

File: ot-jars/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Introduction
+If you read this you probably want to publish a new version of OT jars image.
+In this document you can find instructions how to publish the image.
+
+# Location
+The image is hosted in Docker Hub in sumologic organization https://hub.docker.com/r/sumologic/opentelemetry-jars 
+
+# Version history
+- `v0.0.1` - contains OT jars version `0.3.0`
+- `v0.0.2` - contains OT jars version `0.6.0`
```
Comment (pmm-sumo) [inline ot-jars/README.md:10]: Just a thought - perhaps we could sync release numbers with OT release numbers? So `v0.6.0` or `v0.6.0.0` for the most recent jars?

File: ot-jars/image-build-and-push.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/local/bin/bash
+set -e
+
+VERSION="v0.3.0"
```
Comment (pmm-sumo) [inline ot-jars/image-build-and-push.sh:4]: Version history ends at `v.0.2.0`

File: ot-jars/README.md (hunk 1 above)
Comment (piotr-sumo) [reply ot-jars/README.md:10]: Good idea, I will do that for next releases of OT jars.

File: ot-jars/image-build-and-push.sh (hunk 2 above)
Comment (piotr-sumo) [reply ot-jars/image-build-and-push.sh:4]: 👍

**Pull request merged**